mrc/control/controller.py

```diff
@@ -0,0 +1,20 @@
+from threading import Thread
+from typing import Callable
+
+from mrc.localization.locator import Locator
+
+
+class Controller(Thread):
+    def __init__(self, locator: Locator, dtp_func: Callable([float, float], None)):
```

Comment:
I'm not sure if "dtp_func" will be understandable for users.

Reply:
Name changed